extensions/leveldb/runtime/pom.xml

```diff
@@ -56,6 +57,35 @@
         <dependency>
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-leveldb</artifactId>
+            <exclusions>
+                <exclusion>
+                    <artifactId>leveldbjni-all</artifactId>
+                    <groupId>org.fusesource.leveldbjni</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.camel</groupId>
+            <artifactId>camel-leveldb</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.fusesource.leveldbjni</groupId>
+                    <artifactId>*</artifactId>
+                </exclusion>
```

Comment:
@ppalaga There is only one question. Current PR removes dependency with C++ implementation of leveldDB.
Which makes JVM also using java implementation.
I can return dependency back, but it makes it use by native too, which makes native artifact a little bit bigger.
Do you know about some correct way how to define dependency only for JVM? (which would be the best solution here)

Reply:
How much is little bit bigger? I'd be fine with making the native executable up to 20MB bigger and add the C++ impl for both.

Reply:
Thanks,  I'll check.

Reply:
I've verify the sizes of both artifacts, and there is almost none difference. I expected, that there is some difference, but because jni library is not used in native, quarkus doesn't use it - which makes sense, I've returned leveldbjni into dependencyies for jvm to use,